### ISSUE TITLE
Windows: patches are opened off screen if coordinates exceed screen size

### ISSUE THREAD

**Issue by mxa** [pure-data/pure-data#432]
Workaround is to edit the pd file to bring it back to the screen. However, on Linux this is detected automatically and patches will open on the screen even if their stored window position would be outside of the screen. This doesn't work on win10.

**Comment by umlaeute** [comment]
darn.

**Comment by danomatika** [comment]
@Spacechild1 Do you have time to try this? Tk is supposed to handle bad positions by moving them not the screen and this works in macOS & Linux, but hat may be up to the OS window manager...

**Comment by Spacechild1** [comment]
yeah, I know this issue... I'll try to tackle this, but rather for Pd 0.49.1 :-) right now I'm trying to fix a serious regression with the [donecanvasdialog( message...

**Comment by danomatika** [comment]
I just need someone to confirm and test possible solutions. I added the multi-monitor window positioning handling for the last release.
